src/System.Private.CoreLib/src/System/Reflection/RuntimeConstructorInfo.cs

```diff
@@ -376,13 +376,15 @@ internal void ThrowNoInvokeException()
             return RuntimeMethodHandle.InvokeMethod(obj, null, sig, false, wrapExceptions);
         }
 
+#pragma warning disable CS8609 // TODO-NULLABLE: Covariant return types (https://github.com/dotnet/roslyn/issues/23268)
```

Comment:
The abstract base class annotation for GetMethodBody() should be nullable, and this warning disable should not be needed.

Reply:
Fixed.